app/src/main/res/layout/fragment_edit_news.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <data>
+        <variable
+            name="viewModel"
+            type="com.example.newsapp.ui.editNews.EditNewsViewModel" />
+    </data>
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:gravity="center"
+            android:layout_marginHorizontal="@dimen/m">
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/edit_news"
+                android:textSize="30sp"
+                android:textStyle="bold"
+                android:layout_marginTop="@dimen/m"/>
+
+            <ImageView
+                android:id="@+id/imgGallery"
+                android:layout_width="250dp"
```

Comment:
Do you need to keep fixed dp value?

Reply:
Yes  we need the fixed dp for the image view, for better adjustments for other teammates